systemd/systemd.go

```diff
@@ -328,9 +345,7 @@ func NewEmulationMode(rootDir string) Systemd {
 	if rootDir == "" {
 		rootDir = dirs.GlobalRootDir
 	}
-	return &emulation{
```

Comment:
this was using emulation, not systemd so newSystemd needs a if on kind to pick that

Reply:
Oops! Luckily Maciej went to the rescue :-)